tsconfig.json

```diff
@@ -4,11 +4,9 @@
     "downlevelIteration": true,
     "esModuleInterop": true,
     "inlineSources": true,
-    "importHelpers": true,
```

Comment:
This created a implicit dependency on `tslib` and would fail if it wasn't installed